CONTRIBUTING.md

```diff
@@ -216,6 +216,10 @@ You may have run `yarn build` before. Please run `make clean_all && yarn` to cle
 
 It may also just be a stale cache, you can try deleting the `.parcel-cache`
 
-## Contributor to committer
+## From Contributor To Committer
 
-We love contributions from our community! If you'd like to go a step beyond contributor and become a committer with full write access and a say in the project, you must be invited. The existing committers employ an internal nomination process that must reach lazy consensus (silence is approval) before invitations are issued. If you feel you are qualified and want to get more deeply involved, feel free to reach out to existing committers to have a conversation.
+We love contributions from our community! If you'd like to go a step beyond contributor and become a committer with full write access and a say in the project, you must be invited to the project. The existing committers employ an internal nomination process that must reach lazy consensus (silence is approval) before invitations are issued. If you feel you are qualified and want to get more deeply involved, feel free to reach out to existing committers to have a conversation about that.
```

Comment:
```suggestion
We love contributions from our community! Unfortunately, we are not accepting new committers at this time. We thank you for your understanding.
```

Reply:
@dannify Probably need some more word smithing around this. Anything else we want to say? either to soften or explain?

Reply:
Talked to Danni and she agreed with the change.
Rob, does this soften the language? `At this time, we are not expanding our team of committers. We appreciate your understanding and continued support.`

Reply:
I thought about the `and continued support` as well, or anything along the lines of `we love your contributions`
ultimately it felt too repetitive as it's already stated in the first sentence. I am fine changing it to:

> At this time, we are not accepting new committers. We thank you for your understanding.

Or leaving in the unfortunately. Either way.